geometry_msgs/Pose neatoPosition
---
geometry_msgs/Point32[] points
